delphi/translators/for2py/loop_handle.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python3
+
+"""File: loop_handle.py
+
+Purpose: Read the Fortran AST (obtained from rectify.py) and refactor it
+         to remove the breaks and returns from within loop statements
+
+"""
+
+import sys
+import pickle
+import argparse
+import re
+import json
+import copy
+from typing import Dict
+from delphi.translators.for2py.format import list_data_type
+from delphi.translators.for2py import For2PyError, syntax
+
+
+class RefactorBreaks(object):
+    """This class defines the refactor state of the intermediate AST while
+    removing the breaks and returns inside while loops
+    """
+
+    def __init__(self):
+        self.find_opp = {
+            ".eq.": ".ne.",
+            ".ne.": ".eq.",
+            ".lt.": ".ge.",
+            ".gt.": ".le.",
+            ".le.": ".gt.",
+            ".ge.": ".lt.",
+        }
+        self.return_found = False
+        self.potential_if = dict()
+        self.shifted_body = list()
+        self.new_while_body = list()
+        self.after_return = list()
+        self.new_outer = list()
+        self.tag_level = 0
+        self.is_break = False
+        self.shifted_items = {}
+        self.shifted_level = 0
+
+    def refactor(self, ast: Dict) -> Dict:
+        body = ast["ast"][0]["body"]
+        self.search_while(body)
+
+        # self.search_breaks(body)
+        ast["ast"][0]["body"] = body
```

Comment:
I don't understand lines 47 and 51.

Reply:
This is redundant and not required. Removed line 51.